fpdf/drawing.py

```diff
@@ -75,7 +75,7 @@ class GraphicsStateDictRegistry(OrderedDict):
     A container providing deduplication of graphics state dictionaries across a PDF.
```

Comment:
Unfortunately, I don't fully understand the interaction between drawing.py and the rest of the code, so I can only comment on a few details that caught my eyes.

As always, I reccommend to test every reasonably possible permutation of parameter combinations...